### PR TITLE
Khj 58

### PR DESCRIPTION
**Pull request by Dimon-Mvp**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "react-bootstrap": "^1.0.1",
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.1"
+    "react-scripts": "3.4.1",
+    "redux": "^4.0.5"
```
Comment (fazlmakhsudov) [inline package.json:16]: Have you used redux?

File: src/Billing/BillingDetails/BillingDetails.js [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+        },
+        {
+            'id': 3,
+            'cardNumber': '1234567898765432',
+            'cardName': 'kudsdsdm',
+            'expirationMonth': '15',
+            'expirationYear': '23',
+            'billingAddress': 'Pushkina dom ssdssds'
+        }
+    ]
```
Comment (fazlmakhsudov) [inline src/Billing/BillingDetails/BillingDetails.js:39]: Do you need this extra data in your component?

Comment (Dimon-Mvp) [reply src/Billing/BillingDetails/BillingDetails.js:39]: No, sorry. I will change)

File: src/Billing/BillingDetails/BillingDetails.js [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+        },           
+
+        {'headers': {
+            'Content-Type': 'application/json',
+            'Authorization': `Bearer_${token}`
+        }}).then(response => {
+            // initialization(response.data)
+            setFlag(true)
+        })
+            // .catch(error => {
```
Comment (alenase) [inline src/Billing/BillingDetails/BillingDetails.js:118]: There are a few commented lines, please remove them

**Pull request merged**